src/gui/mzroll/isotopeswidget.cpp

```diff
@@ -323,12 +323,16 @@ void IsotopeWidget::populateByParentGroup(vector<Isotope> masslist, double paren
 		if (!peak)
 			continue;
 
+        vector<mzSample*> samples = {_selectedSample};
+        auto quantity = child.getOrderedIntensityVector(samples,
+                                                        _mw->getUserQuantType()).at(0);
+
 		mzLink link;
 		link.mz1 = parentMass;
 		link.mz2 = child.expectedMz;
 		link.note = isotopeName;
-		link.value1 = child.expectedAbundance;
-		link.value2 = peak->peakIntensity;
+        link.value1 = child.expectedAbundance;
+        link.value2 = quantity;
```

Comment:
`quantity` is a vector but`link.value2` expects a float. Has this been tested?

Reply:
`quantity` is the 0th value in from the intensity vector. Yes this has been tested.

Reply:
Oops. Didn't notice the `.at(0)`, sorry.

But I don't quite understand. The value2 needs to be the intensity for the selected peak. Why take the lowest intensity.

Reply:
Sending only one sample (see line 326), which is the peak's sample. So we are getting the correct quantity.

Reply:
got it. Forgot about that. Guess I shouldn't be reviewing PRs in batches :P